features/step_definitions/signup_steps.rb

```diff
@@ -0,0 +1,19 @@
+Given("User visits the site") do
+  visit root_path
+end
+
+Then("User clicks on {string} button") do |button|
+  click_on(button)
+end
```

Comment:
This is code repetition. See a step definition in this very file with the same capybara command